packages/client3/pages/pools/senior/index.page.tsx

```diff
@@ -186,11 +185,12 @@ export default function SeniorPoolPage() {
         </div>
         <div style={{ gridArea: "info" }}>
           {error ? (
-            <HelperText isError>
-              There was a problem fetching data on the senior pool. Shown data
-              may be outdated.
-            </HelperText>
-          ) : null}
+            <></>
+          ) : // <HelperText isError>
+          //   There was a problem fetching data on the senior pool. Shown data
+          //   may be outdated.
+          // </HelperText>
+          null}
```

Comment:
Old component used by goldfinch. I commented instead of deleting it for future reference.